### DISCUSSION TITLE
Memory usage

### DISCUSSION THREAD

**Discussion by ghost** [zed-industries/zed#8736] [Feature Requests]
How much memory does Zed use when no project is open? Are there any issues related to memory in current releases (I've seen issues about memory leaks)? Does Zed use less RAM than VSCode?

**Comment by Moshyfawn** [comment]
There's an official [performance section](https://arc.net/l/quote/sqmsrdde) on the Zed's landing page that you can refer to _(note the tabs for 'insertion latency', 'boot time' and 'memory footprint' below the graph)_.

**Reply by ghost** [reply]
Thank you!
